prime-router/settings/prod/0011-imagemover-orgs.yml

```diff
@@ -0,0 +1,128 @@
+- name: wi-dph
+  description: Wisconsin Department of Public Health
+  jurisdiction: STATE
+  stateCode: WI
+  receivers:
+    - name: elr
+      organizationName: wi-dph
+      topic: covid-19
+      jurisdictionalFilter:
+        - or(ordering_facility_state, WI, patient_state, WI)
+      translation:
+        type: HL7
+        useBatchHeaders: true
```

Comment:
Is there a reason Wisconsin doesn't have the same fields set as TN, MS and others?